### PR TITLE
collect client requiring tests

### PR DESCRIPTION
**Pull request by zancas**

### CODE REVIEW

File: tests/integration.rs [hunk 1]
```diff
@@ -193,243 +190,247 @@ async fn launch_localnet_lightwalletd_zebrad() {
...
-            100_000,
-            None,
-        )],
-    )
-    .await
-    .unwrap();
-    local_net.validator().generate_blocks(1).await.unwrap();
-    faucet.do_sync(false).await.unwrap();
-    recipient.do_sync(false).await.unwrap();
+mod send {
```
Comment (ala-mode) [inline tests/integration.rs:193]: The only real changes appear to be here and on the following lines, creating a mod, a few needed `use`s, and limiting the "client" feature-gate scope.

File: tests/integration.rs [hunk 2]
```diff
@@ -193,243 +190,247 @@ async fn launch_localnet_lightwalletd_zebrad() {
...
+    #![cfg(feature = "client")]
+    use super::*;
+    use zcash_local_net::client;
+    #[tokio::test]
+    async fn zainod_zcashd_basic_send() {
```
Comment (ala-mode) [inline tests/integration.rs:198]: Here on is changed due to terrible git diffing and indenting. I went through line by line to try to make sure that the removed and added were equivalent, and I think they are.

If anyone knows a great tool for this kind of situation please let me know.

**Review by ala-mode — COMMENTED** [review]
Minimal actual changes, and tests run the same before and after. I'm going to approve a merge.

If anyone knows a good git diffing tool for large diffs that actually aren't diffs (side by side with scrolling even?) please let me know.

**Comment by ala-mode** [comment]
One other point, the parent of this branch seems to be `ffd5997` rather than the merged in `/zingolabs/dev`  at `9c9ce8c`

Other than that, I think this is OK to merge as an incremental change. All tests behave the same with `cargo nextest run` across `ffd5997` `9c9ce8c` and `082bf6b` commits.

**Comment by ala-mode** [comment]
Noticing some breaking changes, particularly with capitalization. And @zancas the branch is still based on `ffd5997` rather than `dev`.

File: tests/integration.rs [hunk 3]
```diff
@@ -921,3 +681,244 @@ mod client_rpcs {
...
+            lightclient_dir.path().to_path_buf(),
+            local_net.indexer().port(),
+        )
+        .await;
+
+        faucet.do_sync(false).await.unwrap();
+        from_inputs::quick_send(
+            &faucet,
+            vec![(
+                &get_base_address(&recipient, pooltype::shielded(shieldedprotocol::orchard)).await,
```
Comment (ala-mode) [inline tests/integration.rs:720]: These lost caps.

File: tests/integration.rs [hunk 4]
```diff
@@ -921,3 +681,244 @@ mod client_rpcs {
...
+                100_000,
+                none,
```
Comment (ala-mode) [inline tests/integration.rs:722]: Also lost caps.

File: tests/integration.rs [hunk 5]
```diff
@@ -921,3 +681,244 @@ mod client_rpcs {
...
+    }
+
+    #[tokio::test]
+    async fn zainod_zebrad_basic_send() {
+        tracing_subscriber::fmt().init();
+
+        let local_net = localnet::<zainod, zebrad>::launch(
+            zainodconfig {
+                zainod_bin: zainod_bin,
+                listen_port: none,
```
Comment (ala-mode) [inline tests/integration.rs:751]: There's a lot of them like this. I'm not going to comment on all of them.

**Review by ala-mode — COMMENTED** [review]
Please see inline notes.

**Comment by zancas** [comment]
@ala-mode thanks for the review.  I caught the caps issues you noticed.   I am closing this PR as superseded by this PR:

#35

**Pull request closed without merging**